<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog</class>
 <widget class="QDialog" name="Dialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>416</width>
    <height>135</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>416</width>
    <height>135</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>416</width>
    <height>135</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>New case...</string>
  </property>
  <layout class="QFormLayout" name="formLayout">
   <item row="3" column="1">
    <widget class="QDialogButtonBox" name="buttonBox">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="standardButtons">
      <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
     </property>
    </widget>
   </item>
   <item row="0" column="1">
    <widget class="QLineEdit" name="case_reference_field">
     <property name="minimumSize">
      <size>
       <width>240</width>
       <height>21</height>
      </size>
     </property>
     <property name="autoFillBackground">
      <bool>false</bool>
     </property>
     <property name="styleSheet">
      <string notr="true"/>
     </property>
     <property name="locale">
      <locale language="English" country="UnitedKingdom"/>
     </property>
     <property name="placeholderText">
      <string>Case reference</string>
     </property>
    </widget>
   </item>
   <item row="1" column="1">
    <widget class="QLineEdit" name="lab_reference_field">
     <property name="minimumSize">
      <size>
       <width>240</width>
       <height>21</height>
      </size>
     </property>
     <property name="locale">
      <locale language="English" country="UnitedKingdom"/>
     </property>
     <property name="placeholderText">
      <string>Lab reference</string>
     </property>
    </widget>
   </item>
   <item row="2" column="1">
    <widget class="QLineEdit" name="investigator_field">
     <property name="minimumSize">
      <size>
       <width>240</width>
       <height>21</height>
      </size>
     </property>
     <property name="locale">
      <locale language="English" country="UnitedKingdom"/>
     </property>
     <property name="placeholderText">
      <string>Investigator name</string>
     </property>
     <property name="clearButtonEnabled">
      <bool>false</bool>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>Dialog</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>248</x>
     <y>254</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>Dialog</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>316</x>
     <y>260</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
